### PR TITLE
Remove brocken symlinks when using copyfiles

### LINKED ISSUE #1886: make copyfile leaves broken artifact links
Issue for PR.

### PR DESCRIPTION
**Pull request by u2mejc**
Resolves #1886

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -56,6 +56,7 @@ copyfiles:
 		rm -rf ${PLUGINS_PATH}/$$plugin && \
 		cp -R plugins/$$plugin ${CORE_PLUGINS_PATH}/available && \
 		ln -s ${CORE_PLUGINS_PATH}/available/$$plugin ${PLUGINS_PATH}/available; \
+                find /var/lib/dokku/ -xtype l -delete;\
```
Comment (michaelshobbs) [inline Makefile]: Fix the indent?

Comment (u2mejc) [inline Makefile]: Opps fixing!

**Comment by u2mejc** [comment]
sowey, I had set my tabs to auto expand.

**Comment by u2mejc** [comment]
lol circleci, you so crazy

**Pull request merged**